app/javascript/packages/components/troubleshooting-options.spec.jsx

```diff
@@ -3,13 +3,24 @@ import TroubleshootingOptions from './troubleshooting-options';
 
 describe('TroubleshootingOptions', () => {
   it('renders a given heading', () => {
-    const { getByRole } = render(<TroubleshootingOptions heading="Need help?" options={[]} />);
+    const { getByRole, getByText } = render(
+      <TroubleshootingOptions heading="Need help?" options={[]} />,
+    );
 
     const heading = getByRole('heading');
 
+    expect(getByText('Need help?').tagName).to.be.equal('H2');
```

Comment:
Could this have used existing `heading` variable? Should be the same?

```suggestion
    expect(heading.tagName).to.be.equal('H2');
```

Reply:
Oops, I moved that test to a different test at the last minute and didn't really notice the heading was already pulled there.